README.md

```diff
@@ -149,23 +150,62 @@ the creation of a new management pack project.
 7. `Enter a path to the Management Pack icon file, or leave blank for no icon`
 
    The icon is used in the VMware Aria Operations UI if present. If it is not present, a default icon will be used. The icon file must be
-   png format and 256x256 px. An icon file can be added later by copying the icon to the root project directory and
+   PNG format and 256x256 pixels. An icon file can be added later by copying the icon to the root project directory and
    setting the value of the `"pak_icon"` key to the icon's file name in the `manifest.txt` file.
 
-8. `Select a language for the adapter. Supported languages are [...]`
-
-   Supported languages are listed. Once selected, the project will be generated, including a template adapter in the
-   selected language. The template adapter collects several objects and metrics from the container that the adapter 
-   is running in, and can be used as a starting point for creating a new adapter.
+[//]: # (TODO: Add this section back when we support it)
+[//]: # (8. `Select a language for the adapter`)
+[//]: # (  selected language. The template adapter collects several objects and metrics from the container that the adapter)
+[//]: # (  Once selected, the project will be generated, including a template adapter in the)
+[//]: # (  is running in, and can be used as a starting point for creating a new adapter.)
+
+For complete documentation of the `mp-init` tool including an overview of its output, see the [MP Initialization Tool Documentation](doc/mp-init.md).
+
+### Template Project
+Every new project creates a file system that has the basic project structure required to develop and build and Management Pack.
+Each file and directory is discussed in depth in the [mp-init](doc/mp-init.md) documentation. `app/adapter.py` is the entry point
+for the your adapter, and the best starting point for a new user to begin their development journey. `adapter.py` is a template
+adapter that has comments throughout its code that allow users to replace the code in the template adapter and turn it into their own adapter. The methods inside the adapter template are all required and should not be deleted; however, the code inside the methods should be substituted to generate the desired adapter. Each method is used for a different function as described below:
```

Comment:
```suggestion
Every new project creates a file system that has the basic project structure required to develop and build and Management Pack.
Each file and directory is discussed in depth in the [mp-init](doc/mp-init.md) documentation. `app/adapter.py` is the entry point
for the your adapter, and is the best starting point. `adapter.py` is a template
adapter that [what does it do?]. It has comments throughout its code that explain what the code in the template adapter does and how to turn it into your own adapter. The methods inside the adapter template are required and should not be deleted; however, the code inside the methods should be modified to generate the desired adapter. Each method is used for a different function as described below:
```

This reads much better than what was there before! I made some small tweaks and a section to fill in. The main thing that I noticed is you were switching between second person (e.g., "your adapter") third person (e.g., "a new user").

Also, while I appreciate the focus on new users, in many cases the same things will apply to more experienced users as well (e.g., starting with 'adapter.py'), so I'd recommend against being more specific than necessary.

Reply:
Also, 'get_adapter_definition' is optional (but recommended - if it's omitted there should be a manually-created `describe.xml` file instead)